blocks/api/raw-handling/index.js

```diff
@@ -88,6 +88,11 @@ export default function rawHandler( { HTML = '', plainText = '', mode = 'AUTO',
 		return parseWithGrammar( HTML );
 	}
 
+	// Normalize unicode to use composed characters.
+	// See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize
```

Comment:
From this link, it is stated there is no support for Internet Explorer:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize#Browser_compatibility

Maybe we want a polyfill?

https://github.com/walling/unorm

Prior art for environment-specific polyfills:

https://github.com/WordPress/gutenberg/blob/270ca2a738fa652f64f1d5a4c01c2d0ea1b1c93c/lib/client-assets.php#L215-L218

Reply:
Hm, thanks for catching that. In this case I'm more tempted to drop the normalisation. What do you think?